src/components/toast/toast.js

```diff
@@ -156,9 +156,30 @@ function MdToastDirective($mdToast) {
   *        <td>`.action(string)`</td>
   *        <td>
   *          Adds an action button. <br/>
-  *          If clicked, the promise (returned from `show()`)
-  *          will resolve with the value `'ok'`; otherwise, it is resolved with `true` after a `hideDelay`
-  *          timeout
+  *          If clicked, the promise (returned from `show()`) will resolve with the value `'ok'`;
+  *          otherwise, it is resolved with `true` after a `hideDelay` timeout.
+  *        </td>
+  *      </tr>
+  *      <tr>
+  *        <td>`.actionKey(string)`</td>
+  *        <td>
+  *          Adds a hot key for the action button. <br/>
+  *          If the `actionKey` and Control are pressed, the toast's action will be triggered.<br>
```

Comment:
Always having this be based on `control` seems a little too specific. The user might want to do `ctrl + shift` or even just a single key. Gmail, for example, uses just `z` for undo. For `MatSnackbar`, we just document that the user should add their own keyboard shortcut.

Reply:
I agree. However this only applies to the "simple" toasts. 

If developers need more flexible options, then the custom toast demo was updated to demonstrate how to setup custom keyboard shortcuts.

Reply:
I'm also happy to extend this API to accept values for the `metaKey`, `ctrlKey`, `shiftKey`, and `altKey` should the request arise.

Do you think that we should just go ahead and support those now?

Reply:
I don't feel super strongly either way.